src/vunnel/providers/mariner/generate_models.py

```diff
@@ -0,0 +1,47 @@
+import os
+import tempfile
+from subprocess import PIPE, Popen
+
+import requests
+
+MARINER_URL_BASE = "https://raw.githubusercontent.com/microsoft/CBL-MarinerVulnerabilityData/main/{}"
+MARINER_URL_FILENAME = "cbl-mariner-{}-oval.xml"
+
+
```

Comment:
there should probably be one of the following about what this is and how to use it:

1. a healthy comment here in this file
2. a new README.md or DEVELOPING.md in this `mariner` dir 
3. a note in the top-level DEVELOPING.md

If you're looking for votes, I think I like the `mariner/DEVELOPING.md` option (2) best. (and if you see a better place shout out)